### ISSUE TITLE
Steps to destroy the demo apps is missing

### ISSUE THREAD

**Issue by AmitKumarDas** [openebs/openebs#187]
The readme & guides available in various demo apps does not talk about destroying the app, volumes, cleanups, etc.

ref - https://github.com/openebs/openebs/tree/master/k8s/demo/<app>

**Comment by AmitKumarDas** [comment]
@Rupeshiya Kubernetes resources are installed by applying the specifications mentioned in each directory at https://github.com/openebs/openebs/tree/master/k8s/demo. Each directory showcases a demo i.e. launching an application using openebs on a kubernetes cluster.

A contributor needs to update the README on the steps to uninstall the application and other resources that were applied as part of the demo. This ensures the kubernetes cluster to be in a clean state.

A contributor is expected to learn kubectl (i.e. kubernetes cli) and operations that can performed by kubectl to install & uninstall kubernetes resources. These inn turn needs to be updated in the README with the particular demo's context.

**Comment by Rupeshiya** [comment]
@hammadmashkoor  resources means here pods, replicasets....?

**Comment by AmitKumarDas** [comment]
Yes... @Rupeshiya These are known as Kubernetes resources (otherwise known as Kind).
